### PR TITLE
Update VTK CI

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

**Comment by mwestphal** [comment]
@Meakk any idea what is going on with cross compiling ? How can I reproduce that locally ?

**Comment by mwestphal** [comment]
SB CI can be seen here: https://github.com/f3d-app/f3d-superbuild/pull/75

File: .github/actions/vtk-install-dep/action.yml [hunk 1]
```diff
@@ -84,6 +84,13 @@ runs:
         sed -i '' '46i\'$'\n''#include "H5CXprivate.h"' ThirdParty/hdf5/vtkhdf5/src/H5Oint.c
         sed -i '' '36i\'$'\n''#include "H5CXprivate.h"' ThirdParty/hdf5/vtkhdf5/src/H5Rint.c
 
+    - name: Install TBB MacOS
```
Comment (Meakk) [inline .github/actions/vtk-install-dep/action.yml:87]: TBB is already installed on Linux docker image but what about Windows? Is it installed by default?

Comment (mwestphal) [reply .github/actions/vtk-install-dep/action.yml:87]: Looks like it is.

Comment (Meakk) [reply .github/actions/vtk-install-dep/action.yml:87]: Ok good to know.

**Review by Meakk — APPROVED** [review]

**Pull request merged**